lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
Sorry, what did you want to accomplish with the rebase? I'm getting `Current branch ref-based-only is up to date.` My `master` branch has edits, so I'm working off of a `source` branch that's a copy of your `master`. That's the message I get on `git rebase source`.